src/lambda-manager/template.yaml

```diff
@@ -246,18 +127,28 @@ Resources:
             Ref: LogsFilter
           REGEX_PATTERN:
             Ref: RegexPattern
-          FIREHOSE_ARN: !GetAtt 
-            - CoralogixDeliveryStream
-            - Arn
-          FIREHOSE_ROLE: !GetAtt 
-            - CloudwatchToFirehoseRole
-            - Arn
+          DESTINATION_ARN: 
+            Ref: DestinationArn
+          DESTINATION_ROLE: 
+            Ref: DestinationRole
+          DESTINATION_TYPE:
+            Ref: DestinationType
+          SCAN_OLD_LOGGROUPS:
+            Ref: ScanOldLogGroups
       Policies:
         - Statement:
+            - Sid: CXLambdaUpdateConfig
+              Effect: Allow
+              Action:
+                - lambda:UpdateFunctionConfiguration
```

Comment:
What is this function supposed to modify in its own/other functions configuration?

Reply:
It needs to modify its own Env Variables.